roles/etcd/handlers/main.yml

```diff
@@ -2,10 +2,50 @@
 - name: restart etcd
   command: /bin/true
   notify:
+    - Backup etcd data
     - etcd | reload systemd
     - reload etcd
     - wait for etcd up
 
+- name: Backup etcd data
+  command: /bin/true
+  notify:
+    - Refresh Time Fact
+    - Set etcd Backup Directory Prefix
+    - Set Backup Directory
+    - Create Backup Directory
+    - Backup etcd v2 data
+    - Backup etcd v3 data
+
+- name: Refresh Time Fact
+  setup: filter=ansible_date_time
+
+- name: Set etcd Backup Directory Prefix
+  set_fact:
+    etcd_backup_prefix: '/var/backups'
```

Comment:
move this to roles/etcd/defaults/main.yml (but keep "Set Backup Directory" task as-is)